### PR TITLE
Increases size of lobby options window

### PR DESCRIPTION
**Pull request by Intigracy**
no more scroll wheels

Ignore the greenish color, that's my windows color for active windows.

### CODE REVIEW

**Comment by clusterfack** [comment]
Make it use browser datums to so it's nice and blue

**Comment by Intigracy** [comment]
how

**Review by Zth-- — APPROVED** [review]

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 	output += "</div>"
 
-	src << browse(output,"window=playersetup;size=210x240;can_close=0")
+	src << browse(output,"window=playersetup;size=210x280;can_close=0")
```
Comment (RemieRichards) [inline code/modules/mob/new_player/new_player.dm]: replace this line with:

```
var/datum/browser/popup = new(src, "playersetup", "New Player Options", 210, 280)
popup.set_content(output)
popup.open()
```

that'll fulfil @clusterfack 's request

Comment (Intigracy) [reply code/modules/mob/new_player/new_player.dm]: Doesn't make it non-closable.

Comment (RemieRichards) [reply code/modules/mob/new_player/new_player.dm]: Ok:

```
new popup blah blah
popup.set_window_options("focus=0;can_close=0;can_minimize=1;can_maximize=0;can_resize=1;titlebar=1;")
popup.open()
```

**Comment by ComicIronic** [comment]
Are the requested changes going to be made?

**Comment by Intigracy** [comment]
Yes.

**Pull request merged**